docs/nodes/README.md

```diff
@@ -19,6 +19,7 @@ node, and then validate the Avalanche network using an AvalancheGo node.
 | [**Run an Avalanche Node with Google Cloud Platform**](./build/set-up-an-avalanche-node-with-google-cloud-platform.md)                      | Create a node that runs on Google GCP       |
 | [**Run an Avalanche Node with Alibaba Cloud**](./build/set-up-node-on-alibaba-cloud.md)     | Create a node that runs on Alibaba Cloud       |
 | [**Run an Avalanche Node with Tencent Cloud**](./build/set-up-node-on-tencent-cloud.md)     | Create a node that runs on Tencent Cloud       |
+| [**Setup an Avalanche Node Using Community Guide**](https://github.com/MoonBoi9001/Avalanche-node-quickstart-quide)     | Comprehensive community made setup guide       |
```

Comment:
Your guide is only for Windows. Please change the title to reflect that.

Reply:
Thanks yulin-dong, I will change it. I'm new to PR's. Is there a way to edit the existing PR, or do I now submit another PR with the changes (would the two PR's automatically merge into one, or does this one get discarded)?

Reply:
Please read doc on git process such as https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request